ocs_ci/helpers/managed_services.py

```diff
@@ -581,6 +582,14 @@ def verify_faas_consumer_resources():
     log.info(f"Client endpoint IP is {ep_ip}")
     assert re.match("\\d+(\\.\\d+){3}", ep_ip)
 
+    # Check that there's no storagecluster on consumer
+    sc_obj = OCP(
+        kind=constants.STORAGECLUSTER,
+        namespace=config.ENV_DATA["cluster_namespace"],
+    )
+    log.info(f"Verifying there's no storagecluster: {sc_obj['items']}")
+    assert sc_obj["items"] == {}
```

Comment:
Fetch the storagecluster resources before verifying.

Reply:
Which resources do you like me to fetch?

Reply:
This is fixed in the recent commit.
```
sc_obj.**get()**
```